frog/imports/client/Wiki/index.js

```diff
@@ -74,7 +74,8 @@ type WikiCompStateT = {
   noInstance: ?boolean,
   username: string,
   isAnonymous: boolean,
-  settings?: WikiSettingsT
+  settings?: WikiSettingsT,
+  isOwner?: boolean
```

Comment:
This should not be a definite type, it is already a truthy/falsy value

Reply:
Made the respective changes